bin/steps/nltk

```diff
@@ -27,6 +27,8 @@ if sp-grep -s nltk; then
         python -m nltk.downloader -d "$BUILD_DIR/.heroku/python/nltk_data" "${nltk_packages[@]}" | indent
         set_env NLTK_DATA "/app/.heroku/python/nltk_data"
 
+        mcount "buildvar.nltk_packages_definition"
+        mcount "steps.nltk"
```

Comment:
Are these two going to always be the same?

Reply:
You are using `steps.vendor.foo` in other cases, but `steps.nltk` here. Is that intentional?

Reply:
Oh, they might be the same, but you can make different queries `buildvar.*` and `steps.*`

Reply:
"vendor" is for anything we're intentionally vendoring. We don't build our own NLTK, just download it, and so it's just a step (like mercurial). 

This is so I can aggregate on multiple dimensions, such as get all `steps`, get all `vendored` steps, get all `buildvars`, or get exactly each one in comparison

Reply:
I missed being consistent on this buildvar output tho - fixing